indico/modules/categories/controllers/display.py

```diff
@@ -97,3 +102,36 @@ def _process_stats(self, stats, root=False):
 
     def _count_users(self):
         return User.find(is_deleted=False, is_pending=False).count()
+
+
+def _serialize_category(category, with_path=False):
+    data = {
+        'id': category.id,
+        'title': category.title,
+        'is_protected': category.is_protected,
+        'category_count': category.deep_children_count,
+        'event_count': category.deep_events_count,
+        'can_access': category.can_access(session.user)
+    }
+    if with_path:
+        data['path'] = [{'id': c.id, 'title': c.title} for c in category.parent_chain_query]
+    return data
+
+
+class RHCategoryInfo(RHDisplayCategoryBase):
+    _category_query_options = undefer('deep_children_count'), undefer('deep_events_count')
```

Comment:
Look at `RHManageCategoryContent._category_query_options` - you need the same thing in there in addition to those two undefers. Right now you don't undefer them for children.

Also, you probably want to subqueryload/joinedload `acl_entries`  since you use `can_access` during serialization